### PR TITLE
Add new utility for converting npz to rds format

### PR DESCRIPTION
**Pull request by barkasn**
This commits adds a new utility for converting npz to rds file format. 

The main utility script is npz2rds.sh, (see --help for options). This script will call the python script npz2txt.py to convert the input data (3 files, one for the sparse matrix and 2 for the columns and row names) into text files. It then calls sparseTxt2RDS.R to convert the text files into an rds files containing a sparse matrix.

Related to issue #477

### CODE REVIEW

File: tools/npz2rds/npz2txt.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+parser = OptionParser()
+parser.add_option('--col-index',dest='colindex', help='column index npy file')
+parser.add_option('--row-index',dest='rowindex', help='row index npy file')
+parser.add_option('--counts',dest='counts', help='counts npz file')
+parser.add_option('--output-dir',dest='output',help='output directory')
+
+(options, args) = parser.parse_args()
+
+## Check that the options are valid
+import numpy as np
```
[the remark this answers is not in the document]
Comment (barkasn) [reply tools/npz2rds/npz2txt.py]: Ok, but I have a specific reason for importing optparse on its own: it ensure that the --help param returns immediately. I don't know if that's a valid exception. In any case I'll fix this now.

[the remark this answers is not in the document]
Comment (rexwangcc) [reply tools/npz2rds/npz2txt.py]: I see your point now. If that's what you want, I'd suggest refactor the scirpt into a function-based script and put the optparse/argparse under:
```
if __name__ == "__main__":
    do optparse/argparse
    do work
```
but I think those improvements could be done in a separate PR.

[the remark this answers is not in the document]
Comment (barkasn) [reply tools/npz2rds/npz2txt.py]: Yea its fine as it is right now, --help is not a performance critical function ;)

File: tools/npz2rds/test/test.sh~ [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```
Comment (rexwangcc) [inline tools/npz2rds/test/test.sh~]: I assume this is a cache file. Could you remove this from the PR?

Comment (barkasn) [reply tools/npz2rds/test/test.sh~]: removed

**Pull request merged**